server/channels/app/export.go

```diff
@@ -171,7 +184,7 @@ func (a *App) BulkExport(ctx request.CTX, writer io.Writer, outPath string, job
 	profilePictures = append(profilePictures, botPPs...)
 
 	ctx.Logger().Info("Bulk export: exporting posts")
```

Comment:
Should the output make it clear that these are only the posts from the exported team?